QUEST.tsv

```diff
@@ -2683,8 +2683,8 @@ QUEST_20151001_002690	But, let's not waste any time.{nl}You've brought the Savio
 QUEST_20151001_002691	I will ask the Savior to help me.{nl}You two should get ready to quickly make the Key of the Night Star.{nl}
 QUEST_20151001_002692	The many deaths of demons could not fill the emptiness in my heart. {nl}Once the goddess's great mission for me is over, {nl}I will be left with a long life full of boring moments.
 QUEST_20151001_002693	Ugh, I thought it would die after its throat had been cut.{nl}I hope it can just behave now and die slowly.
-QUEST_20151001_002694	$A false revelation... all this so you can lure me out to get rid of me?{nl}How disappointing to see that such trickery is the best that Laima can prepare.
-QUEST_20151001_002695	$But for all your preparations, you are powerless against me...{nl}It's amusing how your foolishness blinds you from the overwhelming gap of power between us.
+QUEST_20151001_002694	$A fake revelation.. all this so you can lure me out and get rid of me?{nl}How disappointing to see such trickery is the best that Laima can prepare.
+QUEST_20151001_002695	$But for all your preparations, you are powerless against me..{nl}It's amusing how your foolishness blinds you from the insurmountable gap of power between us.
```

Comment:
I changed it to "overwhelming" because less characters. It makes the dialogue look nice. With this change, the lines will break and look ugly.